src/common/components/private-repos-info/private-repos-info.css

```diff
@@ -4,7 +4,6 @@ $background-color: #FFF;
 
 .info {
   position: relative;
-  margin-bottom: 24px;
 }
 
 .info ul {
```

Comment:
Can we change the ```<ul>``` to be a class? With current styling in Vanilla we try to not style on the base elements but effect them with classes.